Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -79,6 +79,10 @@ final class PaymentViewController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
+    if UIKit.UIDevice.current.userInterfaceIdiom == .phone {
```

Comment:
we should really be using size classes instead of device type for this. Current implementation does not adapt to multitasking on iPad. [See this article on size classes](https://ww-tech.github.io/wwlayout/size-classes.html#:~:text=Size%20classes%20help%20you%20change,regular%20%2C%20compact%20%2C%20and%20unspecified%20.)